buildenv/jenkins/docker-slaves/x86/centOS/Dockerfile

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) 2018, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+
+# To use this docker file:
+#   docker build -t=openj9 .
+#   docker run -it openj9
+
+FROM centos:centos6.9
+
+# Install required OS tools
+
+ENV USER="jenkins"
+
+RUN yum -y install \
+    alsa-lib-devel \
+    autoconf \
+    bind-utils \
+    bison \
+    bzip2 \
+    ca-certificates \
+    cups-devel \
+    cmake \
+    cpio \
+    elfutils-libelf-devel \
+    file-devel \
+    file-libs \
+    flex \
+    fontconfig \
+    fontconfig-devel \
+    freetype-devel \
+    gettext \
+    glibc \
+    glibc-common \
+    glibc-devel \
+    gmp-devel \
+    java-1.7.0-openjdk\
+    java-1.8.0-openjdk\
+    java-1.7.0-openjdk-devel \
+    java-1.8.0-openjdk-devel \
```

Comment:
why so many JDKs are needed? are all of these used?

Reply:
We need 7 to compile 8 (needs to be a full JDK). We need 8 to connect to Jenkins (likely just a jre).
We could use the JDK8 we install below, but 8 below could also be removed since JDK 9 is out of support. I think for now we can just remove the non-devel versions.